threatminer/komand_threatminer/connection/connection.py

```diff
@@ -10,3 +12,14 @@ def __init__(self):
 
     def connect(self, params):
         pass
+
+    def test(self):
+        API_URL = 'https://www.threatminer.org/imphash.php?api=True&rt=2'
+        params = {
+            "q": "1f4f257947c1b713ca7f9bc25f914039"
+        }
+        response = requests.get(API_URL, params=params)
+        if response.status_code != 200:
+            raise Exception(
+                '%s (HTTP status: %s)' % (response.text, response.status_code))
+
```

Comment:
return an empty dict on your test method. If nothing is returned, it could cause issues for the platform.